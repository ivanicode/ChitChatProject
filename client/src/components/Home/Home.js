import React from 'react';

import ChatWindow from '../ChatWindow/ChatWindow';
import MyProfile from '../MyProfile/MyProfile';
import PageNavigation from '../PageNavigation/PageNavigation';
import Profile from '../Profile/Profile';

export function Home(){
    return(
        <div className="page">
            <PageNavigation />
            <MyProfile />
            <ChatWindow />
            <Profile />
        </div>
    )
}

export default Home;
