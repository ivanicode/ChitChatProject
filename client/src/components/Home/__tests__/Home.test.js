import React from 'react';
import {shallow} from 'enzyme';
import { renderHook, act } from '@testing-library/react-hooks';
import {Home, useGetMyContext} from '../Home';

describe('useGetMyContext hook', () => {
    it('should return useContext', () => {
        const hookResult = renderHook(() => useGetMyContext())
    })
})
describe('Home component', () => {
    it('should match snapshot', () => {
        const wrapper = shallow(<Home />)
        expect(wrapper).toMatchSnapshot()
    })
    it('should set parterData when function updateParterData is called with data', () => {
        
    })
})