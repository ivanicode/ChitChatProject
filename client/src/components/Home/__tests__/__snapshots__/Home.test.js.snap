// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should match snapshot 1`] = `
<div
  className="page"
>
  <MyProfile />
  <ChatWindow />
  <Profile />
</div>
`;
